SPECS/libcgroup/libcgroup.spec

```diff
@@ -186,6 +188,9 @@ getent group cgred >/dev/null || groupadd -r cgred
 /tests/gunit/.libs/lt-gtest
 
 %changelog
+* Tue Aug 29 2023 Andy Zaugg <azaugg@linkedin.com> - 2.0.1-24
```

Comment:
It seems we have missed the actual bump to release on line `4`.
IMO, even though we did this change, merging this wont have any impact for a publish.
Since the actual RPM generated would be same as the old one (because nothing apparently changed).

Reply:
As its a change to the package content, i still think we need a minor version bump. I bumped the version.